src/com/theironyard/novauc/Table.java

```diff
@@ -0,0 +1,15 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by dangelojoyce on 2/7/17.
+ */
+public class Table {
+
+    int legs;
+    int screws;
+    String woodType;
+    boolean mahogany;
```

Comment:
Good job D'Angelo.  Naming variables follows the format of lowercase to start the first word uppercase to start the second word.  Use line 10 as a guide